blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import { initialize } from '<%= dasherizedModulePrefix %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+import destroyApp from '../../helpers/destroy-app';
```

Comment:
note that those are the exact same files that are now merged into Ember.js too

see https://github.com/emberjs/ember.js/blob/master/blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js#L4